examples/relative_hydration.py

```diff
@@ -0,0 +1,197 @@
+import simtk.openmm as openmm
+import simtk.unit as unit
+from perses.annihilation.ncmc_switching import NCMCGHMCAlchemicalIntegrator
+import progressbar
+from perses.tests.utils import createSystemFromIUPAC, get_data_filename
+from perses.annihilation.new_relative import HybridTopologyFactory
+from perses.rjmc.topology_proposal import TopologyProposal, SmallMoleculeSetProposalEngine, SystemGenerator
+from perses.rjmc.geometry import FFAllAngleGeometryEngine
+import openeye.oechem as oechem
+import numpy as np
+
+
+kB = unit.BOLTZMANN_CONSTANT_kB * unit.AVOGADRO_CONSTANT_NA
+temperature = 300.0 * unit.kelvin
+kT = kB * temperature
+beta = 1.0/kT
+
+def simulate_hybrid(hybrid_system,functions, lambda_value, positions, nsteps=500, timestep=1.0*unit.femtoseconds, temperature=temperature, collision_rate=5.0/unit.picoseconds):
+    platform = openmm.Platform.getPlatformByName("CUDA")
+    integrator = openmm.LangevinIntegrator(temperature, collision_rate, timestep)
```

Comment:
I think you want to use the `openmmtools.integrators.LangevinIntegrator` or `GHMCIntegrator` instead here.